content/en/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -1,16 +1,20 @@
 ---
 title: kubectl for Docker Users
+content_template: templates/concept
 reviewers:
 - bgrant0607
 - brendandburns
 - thockin
 ---
 
-You can use the Kubernetes command line tool kubectl to interact with the api. You can use kubectl if you are familiar with docker-cli. However, there are a few differences in the docker-cli commands and the kubectl commands. Each of the following section details a docker subcommand and explains the kubectl equivalent.
-
 {{< toc >}}
 
-#### docker run
+{{% capture overview %}}
+You can use the Kubernetes command line tool kubectl to interact with the api. You can use kubectl if you are familiar with docker-cli. However, there are a few differences in the docker-cli commands and the kubectl commands. Each of the following section details a docker subcommand and explains the kubectl equivalent.
+{{% /capture %}}
```

Comment:
- How about removing, `You can use kubectl if you are familiar with docker-cli`. Possibly these edits as well:

You can use the Kubernetes command line tool `kubectl` to interact with the Docker API. However, there are a few
differences between the `docker` commands and the `kubectl` commands. The following sections show a `docker` sub-command and describe the equivalent `kubectl` command.

Reply:
Your suggestion is good. I think "You can use kubectl if you are familiar with docker-cli" is redundancy, too. 
How about edit to the following:

You can use the Kubernetes command line tool kubectl to interact with the API Server. However, there are a few
differences between the docker commands and the kubectl commands. The following sections show a docker sub-command and describe the equivalent kubectl command.

Reply:
@cewuandy, I apologize for the confusion. I think you need some statement indicating that users familiar with the docker-cli will find using kubectl straightforward or easy to use. You can decide whether the command line tool names should be highlight.

How about this:
You can use the Kubernetes command line tool `kubectl` to interact with the API Server. Using `kubectl` is straightforward if you are familiar with the Docker command line tool. However, there are a few differences between the `docker` commands and the `kubectl` commands. The following sections show a `docker` sub-command and describe the equivalent `kubectl` command.

Reply:
@cewuandy I agree with the proposal of @kbhawkey,thanks for your suggestion